examples/visualization/plot_eeglab_head_sphere.py

```diff
@@ -59,9 +59,8 @@
 # the position of Fpz, T8, Oz and T7 channels available in our montage.
 
 # first we obtain the 3d positions of selected channels
-check_ch = ['Oz', 'Fpz', 'T7', 'T8']
-ch_idx = [fake_evoked.ch_names.index(ch) for ch in check_ch]
-pos = np.stack([fake_evoked.info['chs'][idx]['loc'][:3] for idx in ch_idx])
+chs = ['Oz', 'Fpz', 'T7', 'T8']
+pos = np.stack([biosemi_montage.get_positions()["ch_pos"][ch] for ch in chs])
```

Comment:
```suggestion
pos = np.stack([biosemi_montage.get_positions()['ch_pos'][ch] for ch in chs])
```
our stated style preference is for single quotes wherever possible